renovate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": [
+    "config:base",
+    "docker:disable"
+  ],
+  "pinVersions": false,
+  "rebaseStalePrs": true,
+  "schedule": [
+    "after 10pm and before 5am"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "recreateClosed": true,
+    "schedule": [
+      "before 10am on tuesday"
+    ]
+  },
+  "gitAuthor": null
```

Comment:
Nice!  Can you add the `samples` directory to this too?

Reply:
I thought it will pick all `package.json` by default, let's see.

Reply:
Yes, it has:
![image](https://user-images.githubusercontent.com/6311784/41890417-b81b63b0-790f-11e8-9d68-8891c2e9f7b0.png)